readme-vars.yml

```diff
@@ -14,6 +14,21 @@ available_architectures:
   - { arch: "{{ arch_arm64 }}", tag: "arm64v8-latest"}
   - { arch: "{{ arch_armhf }}", tag: "arm32v7-latest"}
 
+# optional block 1
+optional_block_1: true
+optional_block_1_items:
+  - |
+    #### Host vs. Bridge
+
+    Home Assistant can [discover][hb0] and automatically configure
+    [zeroconf][hb1]/[mDNS][hb2] and [UPnP][hb3] devices on your network. In
+    order for this to work you must create the container with `--net=host`.
+
+    [hb0]: https://www.home-assistant.io/integrations/discovery/#mdns-and-upnp
+    [hb1]: https://en.wikipedia.org/wiki/Zero-configuration_networking
+    [hb2]: https://en.wikipedia.org/wiki/Multicast_DNS
+    [hb3]: https://en.wikipedia.org/wiki/Universal_Plug_and_Play
```

Comment:
While Github parses this markdown as expected, does DockerHub?

Reply:
It's standard markdown, but if you know how to test docker hub I can

Reply:
I built the readme locally and manually added it to a private docker hub repo, the links in question all worked